source/install.txt

```diff
@@ -91,21 +91,22 @@ Select the appropriate tab for your operating system:
       Considerations for Ubuntu
       ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-      - ``mongosh`` is available as a
+      - :binary:`mongosh` is available as a
         :abbr:`PPA (Personal Package Archive)` for Ubuntu 20.04 (Focal)
-        and Ubuntu 18.04 (Bionic).
+        and Ubuntu 18.04 (Bionic). If you are installing on Ubuntu
+        20.04 (Focal) or Ubuntu 18.04  (Bionic), use the procedure in
+        the ``.deb`` tab.
         
-        - If you are using Ubuntu 20.04 (Focal) or Ubuntu 18.04
-          (Bionic), use the procedure in the ``.deb`` tab.
-        
-        - If you are using Ubuntu 16.04 (Xenial), you must
-          obtain the ``mongosh`` tarball using the procedure in the
-          ``.tgz`` tab.
+      - MongoDB 5.0 removes support for Ubuntu 16.04 (Xenial). If you
+        must use the unsupported Ubuntu 16.04 (Xenial), install the
+        :binary:`mongosh` tarball using the procedure in the ``.tgz``
+        tab.
       
-      - The ``mongocryptd`` binary in the |mdb-shell| ``.deb`` 
-        package does not work on Ubuntu 16.04 or older. On 
-        Ubuntu 16.04 and older systems, you must 
-        :manual:`install </reference/security-client-side-encryption-appendix/#installation>` 
+      - The ``mongocryptd`` binary that is included in the |mdb-shell|
+        ``.deb`` package does not work on unsupported Ubuntu platforms
+        (16.04 or older). If you must install :binary:`mongosh` on an
+        unsupported Ubuntu platform, :manual:`install
```

Comment:
[suggestion]

When only the "install" text is hyperlinked, it's a little unclear where this link will go. Maybe we should remove the hyperlink from install and add a sentence to this end of this bullet like 

```
To learn how to install ``mongocrptd``, see :manual:`installation
</reference/security-client-side-encryption-appendix/#installation>`
```